homeassistant/components/nina/config_flow.py

```diff
@@ -263,6 +264,10 @@ async def async_step_init(self, user_input=None):
                         CONF_HEADLINE_FILTER,
                         default=self.data[CONF_HEADLINE_FILTER],
                     ): cv.string,
+                    vol.Optional(
```

Comment:
Any particular reason why we store all these in data instead of options?

Reply:
What data do you mean?

Reply:
I noticed we can change many variables from data within the options flow so you can ignore this comment.